ivy_tests/test_ivy/test_frontends/test_jax/test_jax_numpy_indexing.py

```diff
@@ -339,3 +340,70 @@ def test_jax_numpy_unravel_index(
         indices=x[0],
         shape=shape,
     )
+
+
+@handle_frontend_test(
+    fn_tree="jax.numpy.mask_indices",
+    n=helpers.ints(min_value=3, max_value=10),
+    mask_func=st.sampled_from([jax.numpy.triu, jax.numpy.tril]),
+    k=helpers.ints(min_value=-5, max_value=5),
+    input_dtype=helpers.get_dtypes("valid"),
+    test_with_out=st.just(False),
+    number_positional_args=st.just(2),
+)
+def test_jax_numpy_mask_indices(
+    n,
+    mask_func,
+    k,
+    input_dtype,
+    test_flags,
+    frontend,
+    fn_tree,
+    on_device,
+):
+    helpers.test_frontend_function(
+        input_dtypes=input_dtype,
+        test_flags=test_flags,
+        frontend=frontend,
+        fn_tree=fn_tree,
+        on_device=on_device,
+        n=n,
+        mask_func=mask_func,
+        k=k,
+    )
+
+
+@st.composite
+def _get_dtype_square_x(draw):
+    dim_size = draw(helpers.ints(min_value=2, max_value=5))
+    dtype = draw(helpers.get_dtypes("valid"))
+    dtype = [draw(st.sampled_from(tuple(set(dtype))))]
+    x = draw(
```

Comment:
similarly, you do not need to do this kind of extra work here as we already have helper functions that generate arrays with settable size and dtype. like herehttps://github.com/unifyai/ivy/blob/master/ivy_tests/test_ivy/helpers/hypothesis_helpers/array_helpers.py#L190